pkg/cmd/kamelet_add_repo.go

```diff
@@ -49,23 +52,24 @@ func newKameletAddRepoCmd(rootCmdOptions *RootCmdOptions) (*cobra.Command, *kame
 		},
 	}
 
-	cmd.Flags().StringP("operator-id", "x", "camel-k", "Id of the Operator to update.")
+	cmd.Flags().StringP("operator-id", "x", "", "Id of the Operator to update. If not set, the active primary Integration Platform is updated.")
```

Comment:
I'm a bit concerned that the way the flag is used is not consistent with `kamel run` and other commands.
@squakez Do you think it's ok?

Reply:
If it's documented, I guess it's okey.

Reply:
OK, then let's accept it as is.